demos/projects/ESPRESSIF/esp32/main/azure_iot_freertos_esp32_main.c

```diff
@@ -0,0 +1,274 @@
+/* FreeRTOS Real Time Stats Example
+
+   This example code is in the Public Domain (or CC0 licensed, at your option.)
+
+   Unless required by applicable law or agreed to in writing, this
+   software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+   CONDITIONS OF ANY KIND, either express or implied.
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdarg.h>
+#include <string.h>
+
+#include "sdkconfig.h"
+#include "esp_event.h"
+#include "esp_wifi.h"
+#include "esp_wifi_default.h"
+#include "esp_err.h"
+#include "esp_netif.h"
+#include "esp_log.h"
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "freertos/semphr.h"
+#include "nvs_flash.h"
+
+#define TASK_PRIO     3
+
+static uint64_t ulGlobalEntryTime = 1639093301;
+
+static const char *TAG = "example_connect";
```

Comment:
>example_connect [](http://example.com/codeflow?start=26&length=15)

should we name interface as sample_azureiot

Reply:
Fine. Done.